src/app/shared/services/analytics.service.ts

```diff
@@ -0,0 +1,63 @@
+import { Inject, Injectable } from '@angular/core';
+import { DOCUMENT } from '@angular/common';
+import { environment } from '../../../environments/environment';
+import { AppConfigService } from './app-config.service';
+import { Angulartics2GoogleAnalytics } from 'angulartics2/ga';
+import { User } from './user/signup.service';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class AnalyticsService {
+    constructor(@Inject(DOCUMENT) private doc: any,
+                private googleAnalytics: Angulartics2GoogleAnalytics,
+                private config: AppConfigService) {
+    }
+
+    setGATag() {
+        const s = this.doc.createElement('script');
+        s.type = 'text/javascript';
+        s.innerHTML = `(function(i,s,o,g,r,a,m){i['GoogleoogleAnaltyicsAnalyticsObject']=r;i[r]=i[r]||function(){
+    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
```

Comment:
I don't see a second `

Reply:
it's there, in 25:113 ( `ga('create', ...)` row).